### PR TITLE
docs(readme): align with readme template

### PR DESCRIPTION
**Pull request by antonsamper**
#### Reference
- https://financialtimes.atlassian.net/browse/CPP-150

#### Notes
- Aligns README with [README Template](https://github.com/Financial-Times/next/wiki/README-template)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-This will install local git hooks for the repo to help prevent you accidentally
-committing secrets into git.
+```sh
+git clone git@github.com:Financial-Times/next-ci-shared-helpers.git
+cd next-ci-shared-helpers
+make install
+```
+
+
+## Development
```
Comment (andygout) [inline README.md:15]: Did you discover a process that will enable the development process of troubleshooting existing commands and creating new ones?

I wouldn't expect that it is as straightforward as developing code as you would normally, so some guidance might be useful.

Comment (antonsamper) [reply README.md:15]: Good shout! I'm now looking into how to add these helpers project 👍  and will include some examples.

In terms of debugging existing scripts or creating new ones, I'm not too sure what to suggest, most commands require linux so unless you have a vm i cant quite think of how you would do it.

@andygout any suggestions?

Comment (andygout) [reply README.md:15]: Nothing off the top of my head as I've not had much experience writing these sorts of scripts, so I ask out of a curiosity of how to go about it.

It looks like @simonplend contributed to the first version of this repo - I wonder if he has any advice… 😉

Comment (antonsamper) [reply README.md:14]: I'll add an example of how to test scripts locally and also how to use these scripts in other repos

File: README.md [hunk 2]
```diff
@@ -8,12 +8,52 @@ This repo contains a set of common bash scripts to be used in the delivery pipel
...
+### Working locally
+
+Many of the scripts require a Unix environment so in order to run existing scripts you'll need to be familiar with [Docker](https://www.docker.com/get-started).
+
+The first step is to [install docker](https://www.docker.com/get-started) and once you're up and running you'll be able to run the scripts locally inside a Docker container.
+
+For example, the following command will run the `helper-install-awscli` script inside the container.
+
+```shell
+docker run --rm -it -w /tmp --mount type=bind,source="$(pwd)",destination=/tmp circleci/node:8-browsers ./helper-install-awscli
```
Comment (andygout) [inline README.md:25]: What is the expected output? I am trying on my machine and getting a response that does not look like that is intended.

Also might be worthwhile using a more recent Node version as an example to keep our documentation up-to-date.

Comment (antonsamper) [reply README.md:25]: @andygout Sure i'll update it to LTS

The output for the script in the example is the setup/install logs for `awscli`. The last line you should see is:
```
Setting up awscli (1.11.13-1) ...
```

Do you see something else?

**Pull request merged**